src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -96,26 +96,29 @@ public function prepare_items() {
 		}
 
 		$args['posts_per_page'] = $per_page;
-
-		if ( 'scheduled' === $this->tab->get_slug() && ! empty( $_REQUEST['s'] ) ) {
+		$search_term = tribe_get_request_var( 's' );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 12 spaces but found 1 space

<sub>hash: 7df7907f4081a7ae09e4f696c7217124</sub>